frontend/app/src/components/helper/RefreshButton.vue

```diff
@@ -14,36 +14,29 @@ withDefaults(
 const emit = defineEmits<{
   (e: 'refresh'): void;
 }>();
-
-const refresh = () => {
-  emit('refresh');
-};
 </script>
 
 <template>
-  <VTooltip top>
-    <template #activator="{ on, attrs }">
-      <VBtn
-        fab
-        text
-        small
-        depressed
+  <RuiTooltip :popper="{ placement: 'top' }" open-delay="400">
+    <template #activator>
+      <RuiButton
+        icon
+        size="sm"
+        variant="text"
         :disabled="loading || disabled"
-        v-bind="attrs"
         color="primary"
-        @click="refresh()"
-        v-on="on"
+        @click="emit('refresh')"
       >
-        <VProgressCircular
-          v-if="loading"
-          rounded
-          indeterminate
-          size="20"
-          width="2"
+        <RuiProgress
+          v-if="false"
+          variant="indeterminate"
+          circular
+          thickness="3"
+          size="24"
```

Comment:
any reason this is commented? 
and also we can do `:loading="loading"` in the button directly

Reply:
Good catch, I did it to test and missed it,

I am aware, but the transition from the icon to the default loader does not look that good.